metaflow/graph.py

```diff
@@ -179,13 +172,18 @@ def __init__(self, flow):
         self._postprocess()
 
     def _create_nodes(self, flow):
-        module = __import__(flow.__module__)
-        tree = ast.parse(inspect.getsource(module)).body
-        root = [n for n in tree if isinstance(n, ast.ClassDef) and n.name == self.name][
-            0
-        ]
         nodes = {}
-        StepVisitor(nodes, flow).visit(root)
+        for element in dir(flow):
+            func = getattr(flow, element)
+            if hasattr(func, "is_step"):
```

Comment:
```suggestion
            if callable(func) and hasattr(func, "is_step"):
```

This is needed with the new configs since config objects always have is_step as an attribute.